src/users/views.py

```diff
@@ -99,14 +99,15 @@ def post(self, request):
             url = "http://localhost:8000/users/verify?code="
 
         #TODO: Use Sendgrid Templates
-        from_email =  Email("no_reply@example.com")
+        from_email =  Email("no_reply@bquest.ucladevx.com")
         to_email = Email(new_user.email)
         subject = "Pick a Brain with PickMyBruin!"
         content = Content("text/html", 
                 "Thank you for joining BQuest! We are happy you are here! Click the link below to verify that you are a UCLA student, and create your profile. \n" +
                 "You will be receiving emails from us, so make sure to update your address on your profile, if you prefer to use another email.\n"+
-                url+ new_profile.verification_code+
-                "If you have any questions, feel free to contact us on bquest.ucla@gmail.com.")
+                url+ new_profile.verification_code+ 
+                "\nIf you have any questions, feel free to contact us on bquest.ucla@gmail.com.\n"
+                )
```

Comment:
can you have the the close paren on the same indentation level as the open.

Also, for blocks of text, I use 

'/n'.join([
  line1, 
  line2, 
  line3, 
])